python/hail/methods/impex.py

```diff
@@ -382,11 +404,49 @@ def import_locus_intervals(path, reference_genome='default', skip_invalid_interv
     :class:`.Table`
         Interval-keyed table.
     """
-    rg = reference_genome._jrep if reference_genome else None
 
-    t = Env.hail().table.Table.importIntervalList(Env.hc()._jhc, path, joption(rg),
-                                                  skip_invalid_intervals)
-    return Table(t)
+    t = import_table(path, comment="@", impute=False, no_header=True,
+                     types={'f0': tstr, 'f1': tint32, 'f2': tint32,
```

Comment:
str is default, but it's fine to list it out here.

Reply:
I wanted to be explicit about what we were expecting for each field.